pom.xml

```diff
@@ -164,7 +164,12 @@
     <!-- Fester manifestor collections endpoint -->
     <fester.url>http://localhost:9999/collections</fester.url>
 
-    <!-- Bucketeer's batch callback URL: its variables are 'job-name' and 'image-id' -->
+    <!-- The string template of a URL endpoint that updates the batch jobs status queue. This endpoint is updated by
+     AWS Lambda functions and by different Bucketeer instances that have been configured to process images larger
```

Comment:
I wasn't aware that the AWS Lambda function is also using that API endpoint. I think it's good to document this, but I think that because the specific value defined here isn't passed to the lambda function but rather is [defined in that application's configuration](https://github.com/UCLALibrary/kakadu-lambda-converter/blob/c2b5a8f93f1d965187119462389b8b5035909227/pom.xml#L56), the sentence that starts "This endpoint is updated ..." would fit better in the OpenAPI spec in the documentation of `updateBatchJob`.